tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/NewLiferayPluginProjectWizard.java

```diff
@@ -155,66 +157,73 @@ protected void performPostFinish()
         super.performPostFinish();
 
         final NewLiferayPluginProjectOp op = element().nearest( NewLiferayPluginProjectOp.class );
-        final IProject project = CoreUtil.getProject( op.getFinalProjectName().content() );
```

Comment:
Actually I used the method here: final IProject project = CoreUtil.getProject( projectName.getName().content() );